### ISSUE TITLE
[Reproducibility] How to reproduce the performance declared in your paper?

### ISSUE THREAD

**Issue by NK-CS-ZZL** [megvii-research/ECCV2022-RIFE#267]
A simple-yet-effective work! We intend to follow this work. However, we cannot reproduce the performance of the official pre-trained model following the instruction in the 'ReadMe.md'. Specifically, we trained it with the configuration: 
RTX 3090*2, batch_size per GPU 32 (64 in total) , 300 epochs, and execute `python3 -m torch.distributed.launch --nproc_per_node=2 train.py --world_size=2`.  
We notice that in your code, only 95% of the data is used in training. We instead use 100% of the data and still get an inferior result.
Our result in Vimeo 90k test set is "Avg PSNR: 34.90 SSIM: 0.9749", but in the pre-trained, they are 35.615/0.9779. 
So could you give me the detailed pipeline for training? Many thanks for considering my request.

**Comment by hzwer** [comment]
Hi, I noticed that lowering the learning rate may cause a drop in performance, so I adjust it back to 3e-4 now.
But such a low PSNR confuses me because I've had multiple recurrence reports before. Maybe it will help if you give me the train_log file.
Using 95% of the training dataset should not affect the final performance of the model.

**Comment by hzwer** [comment]
I have read your paper, "Towards An End-to-End Framework for Flow-Guided Video Inpainting". You can add my WeChat: hzwer1999

**Comment by NK-CS-ZZL** [comment]
We retrained this code with lr = 3e-4 - 3e-5 and get a reasonable result.
